docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,741 @@
+to verify that, e.g., by pinging, as we demonstrate in the section
+regarding testing a site-to-site connection.
+
+=== "{{gui}}"
+    Currently, there is no way to disable an active connection from the 
+    {{gui}}. If you want to disable an active connection, please use the 
+    OpenStack CLI.
+=== "OpenStack CLI"
+    All you have to do is get on either side of the connection and 
+    disable the VPN connection across the other side. Suppose you are on 
+    the left side of the connection (region `fra1`), and for whatever 
+    reason, you want to disable the site-to-site connection between left 
+    and right (regions `fra1` and `kna1`). First, you might want to 
+    remember the name of the VPN connection to the right:
+        
+    ```bash
+    openstack vpn ipsec site connection list
+    ```
+        
+    ```plain
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    | ID                       | Name             | Peer Address  | Authentication Algorithm | Status |
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    | 5f44be31-0588-4f33-883d- | vpn-conn-to-kna1 | 203.0.113.101 | psk                      | ACTIVE |
+    | 9e2d97be55e1             |                  |               |                          |        |
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    ```
+        
+    That would be `vpn-conn-to-kna1`, and according to the command 
+    output above, it is active. To disable it, type the following:
+        
+    ```bash
+    openstack vpn ipsec site connection set --disable vpn-conn-to-kna1
+    ```
+        
+    Check if it is really disabled:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    It looks like it is disabled --- or `DOWN`.
+        
+    > You will probably have to wait several seconds before seeing a 
+    status change. If you are impatient, try something like this:
+    ```bash
+    watch "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"
+    ```
+    Hit CTRL+C as soon as you see the status change you expect.
+        
+    Now, get on the right side of the connection (region `kna1`), 
+    optionally look for the name of the VPN connection to the left (in our 
+    example, that would be `vpn-conn-to-fra1`), and check its status:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-fra1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+    
+    It turns out that this direction of the connection is also disabled.
```

Comment:
After this (outside the content tab) is where you put the instructions for verifying with `ping`.